Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -251,5 +272,26 @@ void OnDestroy()
             InteractionManager.SourceUpdated -= InteractionManager_SourceUpdated;
             InteractionManager.SourceLost -= InteractionManager_SourceLost;
         }
+
+        public void OnSelect()
+        {
+            ThrowDeprecatedWarning();
```

Comment:
Can't really make the compile errors surface with `[obsolete]` in everyone's code, but I can at least throw the warning every time someone throws `OnSelect`, `OnGazeEnter`, and `OnGazeLeave`.